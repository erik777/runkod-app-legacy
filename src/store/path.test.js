import reducer, {
  selectAct
} from './path';

import {
  selectAct as projectSelect
} from './project';

import {
  logOutAct
} from './user';

let state = undefined;

it('1- default', () => {
  expect(reducer(state, {})).toMatchSnapshot();
});


it('2- select', () => {
  const act = selectAct('/foo/bar/');
  state = reducer(state, act);
  expect(state).toMatchSnapshot();
});

it('3- project select', () => {
  const act = projectSelect({_id: 'projectId'});
  state = reducer(state, act);
  expect(state).toMatchSnapshot();
});


it('3- user logout', () => {
  state = reducer(state, selectAct('/foo/bar/'));

  const act = logOutAct();
  state = reducer(state, act);
  expect(state).toMatchSnapshot();
});