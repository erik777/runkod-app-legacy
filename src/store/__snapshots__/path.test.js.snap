// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default 1`] = `"/"`;

exports[`2- select 1`] = `"/foo/bar/"`;

exports[`3- project select 1`] = `"/"`;

exports[`3- user logout 1`] = `"/"`;
